skimage/restoration/_cycle_spin.py

```diff
@@ -0,0 +1,157 @@
+from concurrent import futures
```

Comment:
Right.  It is available only via a separate `futures` package on 2.7.  I can add a try/catch as you suggest.  However if `dask` offers similar performance I can switch to it.  I just hadn't taken the time to learn it yet :)